package.json

```diff
@@ -54,6 +55,7 @@
         "jest": "29.7.0",
         "jest-extended": "4.0.1",
         "neo4j-driver": "^5.9.2",
+        "prettier": "^3.0.3",
```

Comment:
I notice some of the dependencies are pinned to a certain version and some are using `^`. Is that intentional?

Reply:
Not sure tbh, My best guess is that the pinner dependencies are the things that have been updated by changeset